### PR TITLE
{AppService} Fix linter error in `_github_oauth.py`

### PR DESCRIPTION
**Pull request by jiasli**
**Description**

For unknown reason, Pylint 2.3.0 fails to find linter errors in `_github_oauth.py` added by https://github.com/Azure/azure-cli/pull/17826.

After bumping Pylint 2.8.0 (https://github.com/Azure/azure-cli/pull/17861), those errors are revealed.

https://dev.azure.com/azure-sdk/public/_build/results?buildId=880573&view=logs&j=36dd4138-4d53-5e46-00d9-e5cd9744be05&t=1cf3879c-0a42-5ccd-7693-7a4781d739d8&l=107

```
src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py:20:28: W0613: Unused argument 'cmd' (unused-argument)
```

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 '''
 
 
-def get_github_access_token(cmd, scope_list=None):
+def get_github_access_token(cmd, scope_list=None):  # pylint:disable=unused-import
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py:20]: Since you have removed import, is this change required?

Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py]: I used the wrong checker name. Fixed.

**Comment by yonzhan** [comment]
AppService

**Review by qwordy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 '''
 
 
-def get_github_access_token(cmd, scope_list=None):
+def get_github_access_token(cmd, scope_list=None):  # pylint: disable=unused-argument
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py:20]: How about remove `cmd` parameter directly? As far as I know, custom function is allowed to not contain `cmd` according to https://github.com/Azure/azure-cli/blob/32d269125bec1b051cc35709ea9a631ac8686728/src/azure-cli-core/azure/cli/core/__init__.py#L807-L824
L813 will process smartly

**Review by evelyn-ys — APPROVED** [review]
LGTM in general

File: src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py (hunk 2 above)
Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/appservice/_github_oauth.py:20]: This `default_command_handler` is not used anywhere. Leave it to the author @calvinsID to further decide.
